doc/dev/blkin.rst

```diff
@@ -5,10 +5,17 @@
 Configuring Ceph with LTTng
 ===========================
 
-Use -DWITH_LTTNG option (default: ON)::
+if you compile code, please use -DWITH_LTTNG option (default: ON)::
 
   ./do_cmake -DWITH_LTTNG=ON
 
+if you install ceph visa yum or apt, install the required software packages according to the module which you want to track， otherwise, it may cause a coredump due to missing *tp.solibrary files::
```

Comment:
If your Ceph deployment is package-based (YUM, DNF, APT) vs containerized, install …

Reply:
thank you

Reply:
@anthonyeleven please check it